libs/js-sdk/src/update-strategies/createManualUpdateStrategy.ts

```diff
@@ -44,13 +44,19 @@ export function createManualUpdateStrategy(
         onRequest() {
             return undefined
         },
-        updateAudiences(state, updatedAudiences) {
+        async updateAudiences(state, updatedAudiences) {
+            if (updatedAudiences.sort() === currentAudiences.sort()) {
```

Comment:
I think `.sort` in javascript modifies the array? Might need to create a small utility function